### DISCUSSION TITLE
ゲームから考察するバッジ機能について

### DISCUSSION THREAD

**Discussion by tetsuoikawa** [increments/qiita-discussions#183] [要望・アイデア・不具合]
![image](https://user-images.githubusercontent.com/52817270/159435202-d9e29b3d-3715-4fc6-88af-2116ebb7eef3.png)

バッジ機能について、ゲームなどでよく見かける機能なのですが、この機能をQiitaのバッジ機能にも活かせるのではないかと思い、投稿いたしました。

個人的な意見で申し訳ないのですが

よくゲームなどをやっているときに、バッジというシステムを見かけます。

それで、ゲームにはなんでバッジというシステムがあるのかというのを考えていくと、主に２つの理由があると思いました。

・１つ目が、取るのが難しいバッジを取ったときの達成感

・２つ目が、取ったバッジを他者に見せる事で、自己顕示欲を満たす

ゲームを例に、見ていきます。

１つ目が、取るのが難しいバッジを取ったときの達成感なのですが、ゲームでは、やりこみ要素としてバッジというシステムが存在しています。

取りたい称号（バッジ）を獲得する事を目標にすることで、獲得したときの達成感を味わえることが出来るという感じですね。

基本的には、獲得するのが難しい物ほど人気になる傾向があると思います。（その分達成感が味わえるからだと個人的には思います。）

一つ例として、ApexLegendという、今流行りのゲームを出します。

Apexでは、取るのが難しいバッジとして、　ダブハン　というバッジがあります。

試しに、Youtubeで、apex ダブハンで検索すると、ものすごい数の、ダブハンの取り方講座の動画がヒットします。

それだけ多くの需要があるとともに、多くの人が、ダブハンという取るのが難しいバッジを獲得して、達成感を味わいたいと思っているのではないかと思います。

次に２つ目の、取ったバッジを他者に見せる事で、自己顕示欲を満たすについて。

沢山努力して獲得したバッジ。

試験でいい点数を取った！　難しい資格を取った！

などなど、努力して獲得したものは、他の人に自慢したくなってきます。

ゲームのバッジ機能は、その思いを正確についています。

もう一度、ApexLegendを例にとってみます。

Apexでは、他人のTOPページにいくと、こんな感じで今まで獲得したバッジと、戦績が表示されます。

![image](https://user-images.githubusercontent.com/52817270/159431494-9d5be75a-763c-41b9-9676-4163fd277aa6.png)

Apexにおけるバッジは、こんな形で３つまで設定できるのですが、バッジを設定していない人はほとんどいません。
大抵、何かしらのバッジをつけています。

獲得したバッジを、目につく場所に配置する。

それは、取ったバッジを他者に見せる事で、自己顕示欲を満たす。という事につながるのではないかと思っています。

誠に申し訳ないのですが、そういった観点から見ると、Qiitaにおけるバッジは、獲得したバッジを他人に見せる事が出来ないので、自己顕示欲を満たせないと共に、どんな種類のバッジがあるのかも知られていないので、このバッジが欲しい！と思われる事もあまりないのではないかと思います。

獲得したバッジを、３つ程ピックアップ出来るようにしてして、ピックアップしたやつをHome画面で見れるようにすると共に、どんなバッジの種類があるのかをユーザーにお知らせする事で、バッジ機能がより良い物に発展するのではないかと思います。

長文失礼しました。

**Comment by kskwtnk** [comment]
@tetsuoikawa 
ご意見ありがとうございます！

こちらのDiscussionでも似たようなご意見をいただいていまして、ユーザーページのアップデートを画策しています
https://github.com/increments/qiita-discussions/discussions/166

上記の話と一緒に検討していきます！
